### PR TITLE
[10.0][MIG] stock_available_mrp

### PR DESCRIPTION
**Pull request by SodexisTeam**
P.S. We haven't made modifications for demo/test files.

### CODE REVIEW

File: stock_available_mrp/models/product_product.py [hunk 1]
```diff
@@ -1,62 +1,54 @@
...
+# Copyright 2016-17 Sodexis, Inc. <dev@sodexis.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from collections import Counter
 
-from openerp import models, fields, api
-from openerp.addons import decimal_precision as dp
-
-from openerp.exceptions import AccessError
+from odoo import models, fields, api
```
Comment (rousseldenis) [inline stock_available_mrp/models/product_product.py:8]: Can you change to api, fields, models ?

File: stock_available_mrp/models/product_template.py [hunk 2]
```diff
@@ -1,46 +1,20 @@
...
     @api.depends('potential_qty')
-    def _immediately_usable_qty(self):
+    def _compute_immediately_usable_qty(self):
         """Add the potential quantity to the quantity available to promise.
 
         This is the same implementation as for variants."""
-        super(ProductTemplate, self)._immediately_usable_qty()
+        super(ProductTemplate, self)._compute_immediately_usable_qty()
         for tmpl in self:
             tmpl.immediately_usable_qty += tmpl.potential_qty
```
Comment (rousseldenis) [inline stock_available_mrp/models/product_template.py:20]: See pr #249. There is a performance issue computing quantities using the product field (potential qty) like this.

Prefer compute quantity with modifying dictionary returned by product_available method.

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@SodexisTeam Do you plan to work on this or can be closed as there is a duplicate PR which is quite ready https://github.com/OCA/stock-logistics-warehouse/pull/280/

**Comment by pedrobaeza** [comment]
Closing as this mixes 2 modules and there's another PR opened for the main one.

**Pull request closed without merging**